dev-tools/mage/target/integtest/integtest.go

```diff
@@ -62,12 +62,7 @@ func IntegTest() {
 // Use TEST_COVERAGE=true to enable code coverage profiling.
 // Use RACE_DETECTOR=true to enable the race detector.
 func GoIntegTest(ctx context.Context) error {
-	if !devtools.IsInIntegTestEnv() {
-		mg.SerialDeps(goTestDeps...)
-	}
-	return devtools.RunIntegTest("goIntegTest", func() error {
```

Comment:
Why the change to not run `go test -tags integration` inside of a container? Won't this break some tests expecting containerized services?

Reply:
So far I hit the other problem (for auditbeat):

```
22:28:21 mage goIntegTest
22:28:37 2019/12/17 22:28:37 Found Elastic Beats dir at /go/src/github.com/elastic/beats
22:28:37 >> goIntegTest: Skipping because docker is not available
```